ir/v1.def

```diff
@@ -435,6 +435,7 @@ class AttribLocals : ISimpleNamespace {
     Util::Enumerator<IDeclaration> *getDeclarations() const override {
         return locals.valueEnumerator()->as<const IDeclaration *>(); }
     IDeclaration getDeclByName(cstring name) const override { return locals[name]; }
+    IDeclaration getDeclByName(std::string_view name) const override { return locals[cstring(name)]; }
```

Comment:
Do we want this overload if it just casts to `cstring` anyway? It is probably good for consistency, but could lead to an unexpected cast.

Reply:
Well, we really need this overload since it is an implementation of an interface, without it we will have pure virtual function. `locals` here is `NameMap` – another flavour of string map, which is used only in v1 IR. I decided not to touch it as I'm feeling we do not have enough test coverage for it.